components/UpvoteDownvote.tsx

```diff
@@ -35,9 +35,8 @@ export default function UpvoteDownvote({
   const userFid = user?.fid;
   const castAuthorFid = cast.author?.fid;
   const isSelfPost = userFid === castAuthorFid;
-  const { tip, downvote } = useTipProvider();
```

Comment:
did you also remove the `downvote` method from the `useTipProvider` hook?
- please remove any unused code 🚮